src/utils.c

```diff
@@ -1955,11 +1955,12 @@ static gboolean str_in_array(const gchar **haystack, const gchar *needle)
  *
  * The argument list must be @c NULL-terminated.
  *
+ *
  * @param exclude_vars @c NULL-terminated array of variable names to exclude.
  * @param first_varname Name of the first variable to copy into the new array.
  * @param ... Key-value pairs of variable names and values, @c NULL-terminated.
  *
- * @return The new environment array. Use @c g_strfreev() to free it.
+ * @return @transfer{full} The new environment array. Use @c g_strfreev() to free it.
```

Comment:
`@type{GStrv}` :]

Doesn't returning non-const pointers default to `transfer: full`?